### ISSUE TITLE
Replicate paper results on Traffic Junction env

### ISSUE THREAD

**Issue by abb824** [IC3Net/IC3Net#11]
Hi, 

I am trying to replicate one of the experiments shown in the CommNet article:
> "Impressively, with zero visibility (the cars are driving blind) the CommNet model is still able to **succeed 90%** of the time."

I am trying to run the following:
`python3 main.py --env_name traffic_junction --nagents 10 --nprocesses 16 --num_epochs 2000 --hid_size 128 --detach_gap 10 --lrate 0.001 --dim 6 --max_steps 40 --commnet --vision 0 --add_rate_min 0.05 --add_rate_max 0.02 --curr_start 250 --curr_end 1250 --difficulty medium`

I can't run the code with 16 CPUs (only with 1 CPU), and you mentioned in another issue the results of the paper could only be replicated with 16 CPUs at least, but to what extend?

So I then tried running the following:
`python3 main.py --env_name traffic_junction --nagents 10 --nprocesses 1 --num_epochs 2000 --hid_size 128 --detach_gap 10 --lrate 0.001 --dim 6 --max_steps 40 --commnet --vision 0 --add_rate_min 0.05 --add_rate_max 0.02 --curr_start 250 --curr_end 1250 --difficulty medium`

I get this model summary which seems to make sense:

>   (heads): ModuleList((0): Linear(in_features=128, out_features=2, bias=True))
  (encoder): Linear(in_features=29, out_features=128, bias=True)
  (f_modules): ModuleList((0): Linear(in_features=128, out_features=128, bias=True))
  (C_modules): ModuleList((0): Linear(in_features=128, out_features=128, bias=True))
  (tanh): Tanh()
  (value_head): Linear(in_features=128, out_features=1, bias=True))

But the model performs very poorly (success rate varies between 0.6 and 0.7)

Also, I used the environment with RLlib and I get roughly the same score.

Thanks for any help figuring this out.

**Comment by abb824** [comment]
edited the code so the multiprocessing works.

reopening the issue if the score stays the same.
